handlers.py

```diff
@@ -76,3 +114,5 @@ def handle_connect_wifi(server, body, addr):
     server.send(json.dumps({'type': MsgType.ACK}).encode('utf-8'), addr)
 
     subprocess.run('./shell-scripts/stop-ap.sh')
+
+    carID = register_car(body['user_id'])
```

Comment:
We might want to come up with a different way to trigger this, because at this point the original `Server` object is useless since the AP isn't up anymore. Maybe the original server should close itself here and return `user_id` to `main.py` where the `register_car` function could then be called.

Reply:
True, yeah I forgot about the original Server object. That's a good suggestion, I'll see what we can do.